examples/gesture_recognizer/raspberry_pi/recognize.py

```diff
@@ -0,0 +1,245 @@
+    image = cv2.flip(image, 1)
+
+    # Convert the image from BGR to RGB as required by the TFLite model.
+    rgb_image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
+    mp_image = mp.Image(image_format=mp.ImageFormat.SRGB, data=rgb_image)
+
+    # Run gesture recognizer using the model.
+    recognizer.recognize_async(mp_image, time.time_ns() // 1_000_000)
+
+    # Show the FPS
+    fps_text = 'FPS = {:.1f}'.format(FPS)
+    text_location = (left_margin, row_size)
+    current_frame = image
+    cv2.putText(current_frame, fps_text, text_location, cv2.FONT_HERSHEY_DUPLEX,
+                font_size, text_color, font_thickness, cv2.LINE_AA)
+
+    # Draw the hand landmarks.
+    if recognition_result_list:
+        # Draw landmarks.
+        for hand_landmarks in recognition_result_list[0].hand_landmarks:
+            hand_landmarks_proto = landmark_pb2.NormalizedLandmarkList()
+            hand_landmarks_proto.landmark.extend([
+                landmark_pb2.NormalizedLandmark(x=landmark.x, y=landmark.y,
+                                                z=landmark.z) for landmark in
+                hand_landmarks
+            ])
+            mp_drawing.draw_landmarks(
+                current_frame,
+                hand_landmarks_proto,
+                mp_hands.HAND_CONNECTIONS,
+                mp_drawing_styles.get_default_hand_landmarks_style(),
+                mp_drawing_styles.get_default_hand_connections_style())
+
+    # Expand the frame to show the labels.
+    current_frame = cv2.copyMakeBorder(current_frame, 0, label_padding_width,
+                                       0, 0,
+                                       cv2.BORDER_CONSTANT, None,
+                                       label_background_color)
+
+    if recognition_result_list:
+      # Show top gesture classification.
+      gestures = recognition_result_list[0].gestures
+
+      if gestures:
+        # print(gestures)
+        category_name = gestures[0][0].category_name
+        score = round(gestures[0][0].score, 2)
+        result_text = category_name + ' (' + str(score) + ')'
+
+        # Compute text size
+        text_size = \
+        cv2.getTextSize(result_text, cv2.FONT_HERSHEY_DUPLEX, label_font_size,
+                        label_thickness)[0]
+        text_width, text_height = text_size
+
+        # Compute centered x, y coordinates
+        legend_x = (current_frame.shape[1] - text_width) // 2
+        legend_y = current_frame.shape[0] - (
+                    label_padding_width - text_height) // 2
+
+        # Draw the text
+        cv2.putText(current_frame, result_text, (legend_x, legend_y),
+                    cv2.FONT_HERSHEY_DUPLEX, label_font_size,
+                    label_text_color, label_thickness, cv2.LINE_AA)
+
+      recognition_frame = current_frame
+      recognition_result_list.clear()
+
+    if recognition_frame is not None:
+        cv2.imshow('gesture_recognition', recognition_frame)
+
+    # Stop the program if the ESC key is pressed.
```

Comment:
Can you verify if the escape command works? I've been manually using ctrl+c because esc doesn't seem to work for me. Thanks!

Reply:
I tested it again. The program exits normally with the escape command for me.

Reply:
Weird, alright. I can't get it working on my pi, but that's a later problem.